src/doc/src/reference/manifest.md

```diff
@@ -565,6 +566,25 @@ As for dependents, Cargo suppresses lints from non-path dependencies with featur
 
 > **MSRV:** Respected as of 1.74
 
+## The `[hints]` section
+
+The `[hints]` section allows specifying hints for compiling this crate, which
+crates depending on this one will use by default but may override. Hints are,
```

Comment:
I believe that “crates depending on this one” is an ambiguous or misleading phrase here.

I assume that it actually means that *the workspace/package the `cargo` command was executed on* may override the effect of this hint on this package in its `[profile]`. However, I expect that many will read it to mean that it has some sort of effect that applies to *direct dependents*, which (I assume) it does not.

(Also, both occurrences of “crate” should be “package”, should they not?)